src/library/scala/Console.scala

```diff
@@ -122,7 +122,7 @@ import scala.util.DynamicVariable
  *             a body of code. Threadsafe by virtue of [[scala.util.DynamicVariable]].
  *
  */
-object Console extends DeprecatedConsole with AnsiColor {
+object Console extends AnsiColor {
```

Comment:
I would suggest copying those `final val` into `Console` to cut ties with `scala.io`.